### PR TITLE
include kube-ovn into charm builds

### PR DESCRIPTION
**Pull request by addyess**

### CODE REVIEW

**Review by Cynerva — APPROVED** [review]

**Comment by addyess** [comment]
Do not merge until the following is merged:
* https://github.com/charmed-kubernetes/charm-kube-ovn/pull/9

**Review by mateoflorido — APPROVED** [review]

**Review by Cynerva — APPROVED** [review]

**Pull request merged**
